index.js

```diff
@@ -75,15 +75,16 @@ module.exports = {
       } while (current.parent.parent && (current = current.parent));
     }
 
-    this.buildConfig = target.options['ember-fetch'] || {forcePolyfill: true};
+    this.buildConfig = target.options['ember-fetch'] || { preferNative: false };
```

Comment:
This sounds good. To help forks deal with the upgrade, could we detect if they use `forcePolyfill` property, and throw a useful error message?

Once released, this will also be a major bump.

Reply:
I will squash the commits once I have a more meaningful implementation, right now it's more of a brainstorm